src/chain-spec.ts

```diff
@@ -114,86 +100,109 @@ export async function addBalances(specPath: string, nodes: Node[]) {
-            para_assignment: sr_account.address,
-            beefy: encodeAddress(ec_account.publicKey),
-            aura:
-              chainSessionType == "statemint"
-                ? ed_account.address
-                : sr_account.address,
-          },
-        ];
+  const { sr_stash } = node.accounts;
 
   let keys = getAuthorityKeys(chainSpec);
   if (!keys) return;
 
   keys.push(key);
 
-  // staking
-  if (runtimeConfig?.staking) {
-    runtimeConfig.staking.stakers.push([
+  console.log(
+    `\t👤 Added Genesis Authority ${decorators.green(
+      node.name,
+    )} - ${decorators.magenta(sr_stash.address)}`,
+  );
+
+  writeChainSpec(specPath, chainSpec);
+}
+
+/// Add node to staking
+export async function addStaking(specPath: string, node: Node) {
+  const chainSpec = readAndParseChainSpec(specPath);
+  const runtimeConfig = getRuntimeConfig(chainSpec);
+  if (!runtimeConfig?.staking) return;
+
+  const { sr_stash, sr_account } = node.accounts;
+  runtimeConfig.staking.stakers.push([
+    sr_stash.address,
+    sr_account.address,
+    stakingBond || 1000000000000,
+    "Validator",
+  ]);
+
+  runtimeConfig.staking.validatorCount += 1;
+
+  // add to invulnerables
+  if (node.invulnerable)
+    runtimeConfig.staking.invulnerables.push(sr_stash.address);
+
+  console.log(
+    `\t👤 Added Staking  ${decorators.green(node.name)} - ${decorators.magenta(
       sr_stash.address,
-      sr_account.address,
-      stakingBond || 1000000000000,
-      "Validator",
-    ]);
+    )} - ${stakingBond}`,
+  );
 
-    runtimeConfig.staking.validatorCount += 1;
+  writeChainSpec(specPath, chainSpec);
+}
 
-    // add to invulnerables
-    if (node.invulnerable)
-      runtimeConfig.staking.invulnerables.push(sr_stash.address);
-  }
+/// Add collators
+export async function addCollatorSelection(specPath: string, node: Node) {
+  const chainSpec = readAndParseChainSpec(specPath);
+  const runtimeConfig = getRuntimeConfig(chainSpec);
+  if (!runtimeConfig?.collatorSelection?.invulnerables) return;
 
-  // parachainStaking
-  if (runtimeConfig?.parachainStaking) {
-    runtimeConfig.parachainStaking.candidates.push([
-      eth_account.address,
-      stakingBond || 1000000000000,
-    ]);
-  }
+  const { sr_account } = node.accounts;
 
-  // Collators
-  if (
-    runtimeConfig.collatorSelection &&
-    runtimeConfig.collatorSelection.invulnerables
-  )
-    runtimeConfig.collatorSelection.invulnerables.push(sr_account.address);
+  runtimeConfig.collatorSelection.invulnerables.push(sr_account.address);
 
   console.log(
-    `\t👤 Added Genesis Authority ${decorators.green(
+    `\t👤 Added CollatorSelection  ${decorators.green(
       node.name,
-    )} - ${decorators.magenta(sr_stash.address)}`,
+    )} - ${decorators.magenta(sr_account.address)}`,
   );
 
   writeChainSpec(specPath, chainSpec);
 }
 
+/// Add node to staking
+export async function addParaCustom(specPath: string, node: Node) {
+  /// nop
+}
```

Comment:
Im not sure what this piece of code is. If is temp then maybe add a:
```suggestion
/// Add node to staking
export async function addParaCustom(specPath: string, node: Node) {
  /// nop
  console.info("Not implemented yet");
}
```

Reply:
This is just a `noop` allowing to customize by each `para`. Thanks!